backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/config/ConfigValues.java

```diff
@@ -1581,6 +1581,9 @@ public enum ConfigValues {
     @TypeConverterAttribute(Boolean.class)
     LiveSnapshotPerformFreezeInEngine,
 
+    @TypeConverterAttribute(Boolean.class)
+    LiveStorageMigrationInitialSizeSupported,
```

Comment:
@nirs I wonder about the name, is this one good enough?
Maybe some name that will somehow talk about creating volume with initial size, avoid the need for extending, etc.
What do you think?

Reply:
This means that host support the needsExtend arguemnt in diskReplicateStart() right?

Maybe:

    LiveStorageMigrationExtendedSupported

Or just check for cluster level 4.7?

Reply:
Also, in vdsm we call this feature "replicate_extend":
https://github.com/oVirt/vdsm/commit/f592c261fa3993230a09f40dc24bd05533d7126a

So best name would be:

    ReplicateExtendSupported